src/org/zaproxy/zap/extension/fuzz/httpfuzzer/ui/HttpFuzzResultsContentPanel.java

```diff
@@ -116,6 +122,32 @@ public void itemStateChanged(ItemEvent e) {
 
         toolbar.add(Box.createHorizontalStrut(16));
         toolbar.add(showErrorsToggleButton);
+        
+        /**
+         * Represents the 'Export' button	
+         */
+        toolbar.add(Box.createHorizontalStrut(32));
+        JPanel buttonPanel = new JPanel();   
+        buttonPanel.add(new JButton(new AbstractAction("Export") {
+			private static final long serialVersionUID = 1L;
+
+			@Override
+			public void actionPerformed(ActionEvent e) {
+				JFileChooser chooser = new JFileChooser();
+				if (chooser.showSaveDialog(null) == JFileChooser.APPROVE_OPTION) {

```

Comment:
You might want to leverage org.zaproxy.zap.view.widgets.WritableFileChooser it has some additional functionality for ensuring the destination is actually writable, and handles replacement of existing files.